src/main/resources/TaskManagerCode/Administration.xml

```diff
@@ -20,31 +20,29 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.1">
-  <web>TaskManager</web>
+<xwikidoc>
```

Comment:
You probably did this work locally on an older XWiki instance and resulted in exporting with the old XAR format. I am not sure you want that.

It needs to be redone.